csrc/net/net_module.h

```diff
@@ -3,9 +3,21 @@
 #ifndef MMDEPLOY_SRC_MODULE_NET_MODULE_H_
 #define MMDEPLOY_SRC_MODULE_NET_MODULE_H_
 
+#include <thread>
```

Comment:
> Only keep those that are related to the declaration and put the necessary rest in the definition.

Agree, as we are going to refactor the unit tests in the near future so that some test case will include this header directly, moving all the includes into header file will worsen the build time.

Reply:
done.